ios/Classes/SITFSDKPlugin.m

```diff
@@ -66,8 +68,17 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     } else if ([@"fetchBuildingInfo" isEqualToString:call.method]) {
         [self handleFetchBuildingInfo:call
                                result:result];
-    }else if ([@"getDeviceId" isEqualToString:call.method]) {
+    } else if ([@"getDeviceId" isEqualToString:call.method]) {
         [self getDeviceId:call result:result];
+    } else if ([@"requestNavigation" isEqualToString:call.method]) {
+        [self requestNavigation:call
+                         result:result];
+    } else if ([@"requestDirections" isEqualToString:call.method]) {
+        [self requestDirections:call
+                                   result:result];
+    }else if ([@"stopNavigation" isEqualToString:call.method]){
```

Comment:
Eu cuidaría un pouco máis o formateo do código, dame TOC sólo de ver esto :(

Reply:
jaja, certo. Solucionado